source/2_platform/rex_windows/src/console_application.cpp

```diff
@@ -71,6 +72,8 @@ namespace rex
 
       void update()
       {
+        input::internal::update();
```

Comment:
INFO: Is there a reason why input is stored within the internal namespace?

Reply:
because I didn't want others to run the update, technically they're still able to do so, but it's in an internal namespace which implies that they shouldn't call it

Reply:
But you are calling it?
Is there not a better approach that handle this, can't we PIMPL it?

Reply:
yes, I'm calling it from engine code. we can use pimp if you want to

Reply:
If it does not over complicate things, please do I would use the internal namespace when there is no or no easy solution.